auto_round/script/llm.py

```diff
@@ -192,6 +192,8 @@ def __init__(self, *args, **kwargs):
         self.add_argument(
             "--super_bits", default=None, type=int, help="number of scale and mins quant bits for double quant.")
 
+        self.add_argument("--torch_dtype", default=None, type=str, help="torch_dytpe to load the model.")
```

Comment:
change to another name eval_model_dtype?, as we have a model_dtype in args， and change the discription